src/content/concepts/under-the-hood.md

```diff
@@ -27,7 +28,7 @@ __./app.js__
 export default 'the app';
 ```
 
-By using each other, the modules form a graph (`ModuleGraph`). 
+By using each other, the modules form a graph (`ModuleGraph`).
 
 Usually, modules don't exist by themselves and combine into chunks.
```

Comment:
combine into chunks doesnt make sense without changing to `are getting combined`, but i also feel we could add context and explicitly say that `When outputted by webpack, modules are usually combined into chunks.`

```suggestion
When outputted by webpack, modules are usually combined into chunks.
```

Reply:
Maybe `During bundling, modules are combined into chunks.`?

Reply:
makes sense

Reply:
WDYT?
```suggestion
During the bundling process, modules are combined into chunks.
```

Reply:
👍